public/js/video.js

```
@@ -229,30 +311,126 @@ const VideoChat = (() => {
+          showToast('Microphone enabled', 'success');
+        } catch (err) {
+          if (err.name === 'NotAllowedError' || err.name === 'PermissionDeniedError' || err.name === 'SecurityError') {
+               showToast('Microphone access denied', 'warning');
+          } else {
+               showToast('Microphone error: ' + err.message, 'error');
+          }
+          return;
+        }
+      } else {
+        // Already have a real track, just enable it
+        localStream.getAudioTracks().forEach(t => (t.enabled = true));
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)